pywt/_multidim.py

```diff
@@ -53,46 +57,17 @@ def dwt2(data, wavelet, mode='symmetric'):
 
     """
     data = np.asarray(data)
-    if data.ndim != 2:
-        raise ValueError("Expected 2-D data array")
-
-    if not isinstance(wavelet, Wavelet):
-        wavelet = Wavelet(wavelet)
+    if axes is None:
+        axes = range(-2, 0)
```

Comment:
How about just setting `axes=(-2, -1)` as the default arg? It is more explicit and is also what `np.fft.fft2` uses.